### PR TITLE
Extract fields for PAM.

### PR DESCRIPTION
**Pull request by BerniVarga**

### CODE REVIEW

**Comment by Zapadlo** [comment]
+1

File: extract.go [hunk 1]
```diff
@@ -119,3 +141,12 @@ type appEntry struct {
...
 	Message   string `json:"-"`
 }
+
+type pamEntity struct {
+	UUID string `json:"uuid"`
+	TransactionId string `json:"transaction_id"`
+	PublishDate string `json:"publishDate"`
+	PublishOk string `json:"publishOk"`
+	Duration string `json:"duration"`
+    Endpoint string `json:"endpoint"`
```
Comment (Nesze) [inline extract.go]: go fmt

File: extract.go [hunk 2]
```diff
@@ -19,6 +19,9 @@ var (
...
 	re4 = regexp.MustCompile("([A-Z]{4,5})\\s{1,2}\\[([0-9\\-:,\\s]*)\\] (.*)")
+
+	//[splunkMetrics] 2015/12/21 10:01:37.336610 UUID=08d30fb4-a7b3-11e5-955c-1e1d6de94879 transaction_id=tid_28pbiavoqs publishDate=1450692093737000000 publishOk=true duration=6 endpoint=content
+	pamRegex = regexp.MustCompile("UUID=(.*) transaction_id=(.*) publishDate=(.*) publishOk=(.*) duration=(.*) endpoint=(.*)")
```
Comment (Nesze) [inline extract.go]: (.*) it's a bit greedy

**Comment by Nesze** [comment]
+1

File: extract.go (hunk 2 above)
Comment (Zapadlo) [inline extract.go]: `UUID=([\da-f-]*) transaction_id=(tid_[a-z0-9]*) publishDate=(\d*) publishOk=(\w*) duration=(\d*) endpoint=(\w*)`?

Comment (Nesze) [inline extract.go]: I think something like that would be better.

**Comment by catalin-ilea** [comment]
lgtm

File: extract.go (hunk 1 above)
Comment (BerniVarga) [inline extract.go]: done

File: extract.go (hunk 2 above)
Comment (BerniVarga) [inline extract.go]: Ok, I changed the regex to the one suggested by George. thanks.

**Pull request merged**